lib/LightPanels/LightPanels.js

```diff
@@ -306,7 +355,7 @@ module.exports = kind(
 			this.index = newIndex;
 			this.setupTransitions(currentIndex, false);
 		} else {
-			this.set('index', newIndex);
+			this.set('index', newIndex, true);
```

Comment:
Is it necessary to force a change (now that we've changed the default value of `index` back to `-1`)?